src/controllers/user.controller.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const userService = require('../services/user.service');
+
+const get = (req, res) => {
+  res.statusCode = 200;
+  res.send(userService.getAll());
+};
```

Comment:
for now it is okay, because users is always exist, but in a real project you should handle error here, because you will take users from db etc.

Reply:
thanks, I will handle it in the next task